README.md

```diff
@@ -4,7 +4,6 @@
 
 [![HTML Manuscript](https://img.shields.io/badge/manuscript-HTML-blue.svg)](https://manubot.github.io/rootstock/)
 [![PDF Manuscript](https://img.shields.io/badge/manuscript-PDF-blue.svg)](https://manubot.github.io/rootstock/manuscript.pdf)
-[![Build Status](https://travis-ci.com/manubot/rootstock.svg?branch=master)](https://travis-ci.com/manubot/rootstock)
```

Comment:
I like the suggestion to include a GitHub Actions badge here.  Because we can't adaptively show the Travis or GitHub Actions badge based on which CI service is being used, would we default to GitHub's?

Reply:
I think it's okay to have both. Travis CI fails gracefully

![](https://api.travis-ci.com/manubot/doesnotexist.svg?branch=master)

GitHub's doesn't. Note I'll have to merge to even see whether the GitHub badge works.